### PR TITLE
Update NOTES.txt

### PR DESCRIPTION
**Pull request by consideRatio**
This PR rephrases and makes suggested command a bit less magical to help the user learn some relevant things instead of relying on being provided magic commands. Otherwise, its mostly style changes. This is how it looks when installed without any changes to the chart's default values. If ingress or node ports were configured etc, it would look a bit different.

```
NOTES:
.      __                             __                   __  __            __
      / /  __  __    ____    __  __  / /_  ___    _____   / / / /  __  __   / /_
 __  / /  / / / /   / __ \  / / / / / __/ / _ \  / ___/  / /_/ /  / / / /  / __ \
/ /_/ /  / /_/ /   / /_/ / / /_/ / / /_  /  __/ / /     / __  /  / /_/ /  / /_/ /
\____/   \__,_/   / .___/  \__, /  \__/  \___/ /_/     /_/ /_/   \__,_/  /_.___/
                 /_/      /____/

       You have successfully installed the official JupyterHub Helm chart!

### Installation info

  - Kubernetes namespace: jupyterhub
  - Helm release name:    jupyterhub
  - Helm chart version:   2.0.0-alpha.1
  - JupyterHub version:   2.0.0b1
  - Hub pod software:     See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/2.0.0-alpha.1/images/hub/requirements.txt

### Followup links

  - Documentation:  https://z2jh.jupyter.org
  - Help forum:     https://discourse.jupyter.org
  - Social chat:    https://gitter.im/jupyterhub/jupyterhub
  - Issue tracking: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues

### Post-installation checklist

  - Verify that created Pods enter a Running state:

      kubectl --namespace=jupyterhub get pod

    If a pod is stuck with a Pending or ContainerCreating status, diagnose with:

      kubectl --namespace=jupyterhub describe pod <name of pod>

    If a pod keeps restarting, diagnose with:

      kubectl --namespace=jupyterhub logs --previous <name of pod>

  - Verify an external IP is provided for the k8s Service proxy-public.

      kubectl --namespace=jupyterhub get service proxy-public

    If the external ip remains <pending>, diagnose with:

      kubectl --namespace=jupyterhub describe service proxy-public

  - Verify web based access:

    You have not configured a k8s Ingress resource so you need to access the k8s
    Service proxy-public directly.

    If your computer is outside the k8s cluster, you can port-forward traffic to
    the k8s Service proxy-public with kubectl to access it from your
    computer.

      kubectl --namespace=jupyterhub port-forward service/proxy-public 8080:http

    Try insecure HTTP access: http://localhost:8080
```

### Notes about the text

- The ASCII art is generated via https://patorjk.com/software/taag/#p=display&h=0&f=Slant&t=JupyterHub
- The initial `.` as the absolute first character is needed to avoid a whitespace trim made by the helm CLI
- The git reference used in the link to Hub pod software will work also for dev releases
- I added a link to gitter as "social chat"

### CODE REVIEW

File: jupyterhub/templates/NOTES.txt [hunk 1]
```diff
@@ -1,54 +1,90 @@
...
 
- kubectl --namespace={{ .Release.Namespace }} get pod
+### Installation info
 
-and watching for both those pods to be in status 'Running'.{{ println }}
+  - Kubernetes namespace: {{ .Release.Namespace }}
+  - Helm release name:    {{ .Release.Name }}
+  - Helm chart version:   {{ .Chart.Version }}
+  - JupyterHub version:   {{ .Chart.AppVersion }}
+  - Hub pod software:     See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/{{ include "jupyterhub.chart-version-to-git-ref" .Chart.Version }}/images/hub/requirements.txt
```
Comment (yuvipanda) [inline jupyterhub/templates/NOTES.txt:18]:
```suggestion
  - Hub pod packages:     See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/{{ include "jupyterhub.chart-version-to-git-ref" .Chart.Version }}/images/hub/requirements.txt
```

`software` feels too generic?

File: jupyterhub/templates/NOTES.txt [hunk 2]
```diff
@@ -1,54 +1,90 @@
-Thank you for installing JupyterHub!
+{{- $proxy_service := include "jupyterhub.proxy-public.fullname" . -}}
 
-Your release is named "{{ .Release.Name }}" and installed into the namespace "{{ .Release.Namespace }}".
+.      __                             __                   __  __            __
```
Comment (yuvipanda) [inline jupyterhub/templates/NOTES.txt:3]:
```suggestion
{# Generated with https://patorjk.com/software/taag/#p=display&h=0&f=Slant&t=JupyterHub #}
.      __                             __                   __  __            __
```

File: jupyterhub/templates/NOTES.txt [hunk 3]
```diff
@@ -1,54 +1,90 @@
-Thank you for installing JupyterHub!
+{{- $proxy_service := include "jupyterhub.proxy-public.fullname" . -}}
```
Comment (yuvipanda) [inline jupyterhub/templates/NOTES.txt:1]: I see that we use snake case for local go template variables elsewhere in z2jh too! I thought we were doing camelCase but I was wrong!

**Review by yuvipanda — APPROVED** [review]
THIS IS AWESOME!

Minor comments, but happy to merge after

File: jupyterhub/templates/NOTES.txt (hunk 2 above)
Comment (consideRatio) [reply jupyterhub/templates/NOTES.txt:3]: @yuvipanda I made this to be a Helm template comment syntax wise, was that your intention? I think it is what makes most sense for this.

**Comment by yuvipanda** [comment]
Thanks a lot, @consideRatio!

**Pull request merged**